content/en/docs/tasks/debug-application-cluster/debug-service.md

```diff
@@ -246,20 +291,21 @@ passed into `kubelet` with the `--cluster-dns` flag.
 
 The `search` line must include an appropriate suffix for you to find the
 `Service` name.  In this case it is looking for `Services` in the local
-`Namespace` (`default.svc.cluster.local`), `Services` in all `Namespaces`
-(`svc.cluster.local`), and the cluster (`cluster.local`).  Depending on your own
-install you might have additional records after that (up to 6 total).  The
-cluster suffix is passed into `kubelet` with the `--cluster-domain` flag.  We
-assume that is "cluster.local" in this document, but yours might be different,
-in which case you should change that in all of the commands above.
+`Namespace` ("default.svc.cluster.local"), `Services` in all `Namespaces`
+("svc.cluster.local"), and lastly for names in the cluster ("cluster.local").
+Depending on your own install you might have additional records after that (up
+to 6 total).  The cluster suffix is passed into `kubelet` with the
+`--cluster-domain` flag.  We assume that is "cluster.local" in this document,
+but yours might be different, in which case you should change that in all of
+the commands above.
 
 The `options` line must set `ndots` high enough that your DNS client library
 considers search paths at all.  Kubernetes sets this to 5 by default, which is
 high enough to cover all of the DNS names it generates.
 
-### Does any Service exist in DNS?
+### Does any Service work by DNS name?
```

Comment:
This change would break inbound links. To preserve them, try:
```suggestion
### Does any Service work by DNS name? {#does-any-service-exist-in-dns}
```

Reply:
ACK